src/wallet/wallet.cpp

```diff
@@ -860,6 +860,22 @@ bool CWallet::IsSpentKey(const uint256& hash, unsigned int n) const
     return false;
 }
 
+void CWallet::SetUsedAddressState(WalletBatch& batch, const CTxOut& cout, uint256 hash)
```

Comment:
```suggestion
void CWallet::SetUsedAddressState(WalletBatch& batch, const CTxOut& cout, const uint256& hash)
```

Reply:
Thanks @jonatack, this should now be resolved in bb4790816d84f850367c99134ea6a16659fbb180